src/components/do-dont/DoDont.tsx

```diff
@@ -0,0 +1,24 @@
+import { GoAIcon } from "@abgov/react-components";
+import "./DoDont.css";
+
+type DoDont = "do" | "dont";
+interface Props {
+    type: DoDont;
+    content: string;
```

Comment:
The `content` will need to be a `ReactNode` to allow other non-string elements to be displayed ex images, rendered components, etc